src/sentry/api/analytics.py

```diff
@@ -0,0 +1,29 @@
+from __future__ import absolute_import, print_function
+
+from sentry import analytics
+
+
+class OrganizationSavedSearchCreatedEvent(analytics.Event):
+    type = 'organization_saved_search.created'
+
+    attributes = (
+        analytics.Attribute('organization_id'),
+        analytics.Attribute('search_type'),
+        analytics.Attribute('id'),
+        analytics.Attribute('user_id'),
+    )
+
+
+class OrganizationSavedSearchDeletedEvent(analytics.Event):
+    type = 'organization_saved_search.deleted'
+
+    attributes = (
+        analytics.Attribute('organization_id'),
+        analytics.Attribute('search_type'),
```

Comment:
Had to change  `analytics.Attribute('type')` to  `analytics.Attribute('search_type')`. As it is written `analytics.Event` utilizes a `type` passed in for the `Event` type instead of simply viewing it as an attribute. changing it to `search_type` fixed this issue.